packages/start-frontend/__tests__/cli.test.ts

```diff
@@ -13,9 +13,9 @@ const KEY = {
 };
 
 // Timeout duration for interactive tests, to allow for code stub downloads
-const INTERACTIVE_TEST_TIMEOUT = 10000;
+const INTERACTIVE_TEST_TIMEOUT = 20000;
```

Comment:
I'm sorry I haven't been keeping up lately, but did increasing the timeout make a difference? I think it would be ideal to have as low a timeout as functionally possible, since it would reduce the overall run time.

Reply:
Ah, that's leftover of testing.
I'll revert back.

Reply:
Nvm, there is another issue stopped CLI therefore timeout is happened.
I've revert back to 10000